src/app/pages/questions/components/question/audio-input/audio-input.component.ts

```diff
@@ -71,11 +71,8 @@ export class AudioInputComponent implements OnDestroy, OnInit {
     if (!this.isRecording()) {
       this.recordAttempts++
       if (this.recordAttempts <= DefaultMaxAudioAttemptsAllowed) {
-        this.startRecording()
-          .then(() =>
-            this.usage.sendGeneralEvent(UsageEventType.RECORDING_STARTED)
-          )
-          .catch(e => this.showTaskInterruptedAlert())
+        this.usage.sendGeneralEvent(UsageEventType.RECORDING_STARTED)
```

Comment:
you can also put this in startRecording funtion just before recording starts

Reply:
okay, made the changes.